sync/sync.go

```diff
@@ -42,6 +53,9 @@ func (s *Sync) SyncServices() error {
 }
 
 func (s *Sync) syncServices() error {
+	if check, err := s.shouldPerformSync(); !check {
```

Comment:
Probably it's out of scope of this PR but I've just noticed we are not handling this error and any error generated by [`SyncServices`](https://github.com/allegro/marathon-consul/pull/20/files#diff-0c2cc759dcd70bc815a12c3ee5ce3270R38)